server/models/tags/getTag.js

```diff
@@ -0,0 +1,26 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {Object} param0.query
+ * @return {Promise}
+ */
+
+function getTag({ query }) {
+  return new Promise((resolve, reject) => {
+    const keyword = query.keyword || ''
+    pool.query(
+      `SELECT * FROM tags WHERE id = ? OR name LIKE ?`,
```

Comment:
@NBNARADHYA this will handle both id and name

Reply:
I think it is better to add a param called `id` instead of doing this, because it is highly unlikely that the exact `id is` set to `search` param

Reply:
remove the `id` thing. Why would one need to search a tag by id ?
I mean, people will generally, search a tag by `name`..

Reply:
We can later add the `id` thing, if required. Is it fine ?